File: php/Car.php
<?php
require_once 'Account.php';
class Car
{
    public $id;
    public $license;
    public $driver;
    public $passenger;

    public function __construct($license, $driver){
        $this->$license = $license;
        $this->$driver = $driver;
    }

    public function printDataCar(){
        
        echo '<p> License: ' . $this->license . "</p>";
        echo '<p> Driver Name: ' . $this->driver->name . "</p>";

    }
}
?>

File: java/Car.java
class Car {
    
    private Integer id;
    private String license;
    private Account driver;
    private Integer passenger;

    public Car(String license, Account driver){

        this.license = license;
        this.driver  = driver;

    }

    void printDataCar(){

        if(passenger != null){
        System.out.println("License: " + license + " Name Driver: " + driver.name + " passenger: " + passenger );
        }

    }

    //METODOS SETTER & GETTER

    //@Passenger
    public Integer getPassenger(){
        return passenger;
    }

    public void setPassenger(Integer passenger){

        if(passenger != null){
            this.passenger = passenger;
        } else {
            System.out.println("Necesitas aginar pasajeros");
        }
        
    }

    //@id
    public Integer getId(){
        return id;
    }

    public void setId(Integer id){
        this.id = id;
    }

    //@License
    public String getLicense(){
        return license;
    }

    public void setLicense(String license){
        this.license = license;
    }

    //@driver
    public Account getDriver(){
        return driver;
    }

    public void setDriver(Account driver){
        this.driver = driver;
    }
    
}